http/http-minimum-reactive/src/main/java/io/quarkus/ts/http/minimum/reactive/JakartaRestInterceptor.java

```diff
@@ -0,0 +1,30 @@
+package io.quarkus.ts.http.minimum.reactive;
+
+import java.io.IOException;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.ws.rs.WebApplicationException;
+import jakarta.ws.rs.ext.Provider;
+import jakarta.ws.rs.ext.ReaderInterceptor;
+import jakarta.ws.rs.ext.ReaderInterceptorContext;
+
+import org.jboss.logging.Logger;
+
+/**
+ * The type Jakarta rest workflow tracker.
+ */
+@Provider
+@ApplicationScoped
+public class JakartaRestInterceptor implements ReaderInterceptor {
+
+    private static final Logger LOG = Logger.getLogger(JakartaRestInterceptor.class);
+
+    @Override
+    public Object aroundReadFrom(ReaderInterceptorContext context)
+            throws IOException, WebApplicationException {
+        LOG.info("Before reading " + context.getGenericType());
```

Comment:
IMHO we don't need logging here

Reply:
We didn't, but now I've added checks for the logs

Reply:
ok